float32 distance_setpoint
float32 yaw_setpoint
